### ISSUE TITLE
windows client crashed when i press the log in button

### ISSUE THREAD

**Issue by Starcrackeriii** [tailscale/tailscale#8974]
### What is the issue?

---------------------------
Tailscale Error
---------------------------
Failed to connect to Tailscale service.

Version 1.48.0 (t3f9a4eddb-g6d37ccfec)

LogID: 7c0743a98c2baf6bda8bc75283b51032bf84412730c7ad3a985434882690861e

(Ctrl-C to copy this text)

### OS

Windows

**Comment by bradfitz** [comment]
Is this machine behind a restrictive firewall?

Any other networking/antivirus software running on the machine from the list https://github.com/tailscale/tailscale/wiki/OtherSoftwareInterop or similar?

**Comment by bradfitz** [comment]
@dblohm7, from those frontend logs it does look like the service was at least briefly running, but we don't know its log ID to see what it was doing.

**Comment by zpengcom** [comment]
I'm sure Tailscale doesn't work when I run **Clash for Windows**: can't connect to the network, and clicking Login doesn't bring up the login page; when I close Clash for Windows, rerun Tailscale and it works again, even when I start Clash for Windows will not have any effect

**Comment by bradfitz** [comment]
@Starcrackeriii, is that your problem too? Are you running Clash?

**Comment by Starcrackeriii** [comment]
@bradfitz No, actrually it worked well at the beginning but after i reinstalled it, here comes to the problem

**Comment by liangyoufeng** [comment]
I also encountered the same problem. After reinstalling, I found that the tailscale driver in Windows Computing Section Management Device Manager repeatedly installed and uninstalled. I couldn't tell from the driver log what caused it. I tried resetting the network driver, uninstalling the tailscale program, restarting the system, and then reinstalling, but this error still occurred. But after reinstalling the Windows system, it can be used normally again, and then uninstalling, the error occurs again during reinstallation. I checked the issue and many people have raised this issue, which is not the case on Linux.
